Sources/XCTest/XCTAssert.swift

```diff
@@ -267,7 +267,7 @@ public func XCTAssertNil(@autoclosure expression: () -> Any?, @autoclosure _ mes
         if value == nil {
             return .Success
         } else {
-            return .ExpectedFailure("\"\(value)\"")
+            return .ExpectedFailure("\"\(value!)\"")
```

Comment:
I had initially chosen to not reorder it that way because I liked the symmetry of the conditionals between the various assertion functions, but I have no problem flipping it.